README.md

```diff
@@ -27,26 +27,66 @@ export LIBCLANG_PATH=/usr/lib/clang/4.0/lib
 ```
 
 ### Windows
-
 1. Download and unzip [MozTools 4.0](https://github.com/servo/servo-build-deps/releases/download/msvc-deps/moztools-4.0.zip).
 
-2. Download and install Clang for Windows (64 bit) from <https://releases.llvm.org/download.html>.
-
-3. Open up a shell configured to use Visual Studio. This could be the
-   one included with Visual Studio (e.g. Visual Studio 2017 / X64 Native
-   Tools Command Prompt for VS 2017) or a shell in which you have run:
+2. Download and install Clang for Windows (64 bit) from <https://releases.llvm.org/download.html>. 
```

Comment:
Do we want to suggest downloading LLVM through Visual Studio? It's what we use in CI after all.

Reply:
I have no idea, I and my colleagues are downloading the most recent manually.
I uses theird party bin
he is using scoop